RedHatInsights/cloud-connector
# Do not send reconnect message on org id lookup failures

[ ] Test this manually
- make sure a tenant-less connection can be updated on the next message received
- make sure tenant-less connections cannot be located by api 
- make sure tanant-less connections can be unregistered correctly
[ ] Modify "relaxed" connection locator to verify that the org_id is not empty (add a test here)
[ ] Add a test that verifies that a valid tenant can update a tenant-less connection
[ ] Add a test that verifies that a tenant-less connection can update a valid tenant connection

## What?
Explain what the change is linking any relevant JIRAs or Issues.

## Why?
Consider what business or engineering goal does this PR achieves.

## How?
Describe how the change is implemented. Any noteable new libaries, APIs, or features.

## Testing
Did you add any tests for the change?

## Anything Else?
Any other notes about the PR that would be useful for the reviewer. 

## Secure Coding Practices Checklist Link
- https://github.com/RedHatInsights/secure-coding-checklist

## Secure Coding Checklist
